src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
```

Comment:
All magic numbers in your code should be constants. If there is a default value for the side of the square, it should be defined as a constant.